bpf/bpf_lxc.c

```diff
@@ -685,9 +687,11 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 		else
 			return ret;
 	}
-#else
-	goto pass_to_stack;
 #endif
+	if (is_defined(ENABLE_REDIRECT_NEIGH))
+		return redirect_neigh(ct_state.ifindex ? :
+				      DIRECT_ROUTING_DEV_IFINDEX, 0);
```

Comment:
What's going to happen with a packet which dst is world when a non-direct routing iface is connected to world?